cql3/Cql.g

```diff
@@ -1868,6 +1885,7 @@ basic_unreserved_keyword returns [sstring str]
         | K_CLUSTERING
         | K_COMPACT
         | K_STORAGE
+        | K_TABLES
```

Comment:
You'll also need to add (here and below) K_TYPES, K_CLUSTER, K_INTERNALS K_ONLY, K_AGGREGATES. This is what the Cassandra patch added (https://github.com/apache/cassandra/commit/4e47bfb3a1abb8074fb9a24f98a97dbf25806522)

Reply:
added

Reply:
@Jadw1 ?

Reply:
K_ONLY is already added. However when I'm adding K_SCHEMA I'm getting warning
```
Decision can match input such as "K_SCHEMA" using multiple alternatives: 2, 15
```
If I remove K_ONLY from there and only add K_SCHEMA, there is no warning, so my guess is this is because of:
https://github.com/scylladb/scylladb/pull/11106/commits/b3064adaf9d2ed038f6b6ef9b26e316c28fe5dc3#diff-0a5086285c82b892e8df1e10d947cde809a42fc8b1b947a053cba3315fe779c8R1402

I didn't manage to fix it. Note that in Cassandra's grammar, also only K_ONLY is present in `basic_unreserved_keywords`
https://github.com/apache/cassandra/blob/bf96367f4d55692017e144980cf17963e31df127/src/antlr/Parser.g#L1843-L1904